src/queueloader.cpp

```diff
@@ -78,6 +81,22 @@ void queueloader::reload(std::vector<download>& downloads, bool remove_unplayed)
 					utils::tokenize_quoted(line);
 				bool url_found = false;
 
+				if (fields.size() == 0) {
+					if (!comments_ignored) {
+						std::cout << strprintf::fmt(
+							     _("WARNING: Comment found "
+							       "in %s, the queue file is regenerated "
+							       "when podboat exists and comments will "
+							       "be deleted. Press enter to continue or "
+							       "Ctrl+C to abort"),
+							     queuefile.c_str())
```

Comment:
`strprintf::fmt` understands `std::string`, so please remove that `.c_str()` call.